app/models/course/discussion/topic.rb

```diff
@@ -10,6 +12,14 @@ class Course::Discussion::Topic < ActiveRecord::Base
 
   accepts_nested_attributes_for :posts
 
+  # Topics to be displayed in the comments center.
```

Comment:
Are we prefer UK english ?  sorry most of time I cannot differentiate US and UK english ><